documentation/sphinx/source/api-version-upgrade-guide.rst

```diff
@@ -9,9 +9,9 @@ This document provides an overview of changes that an application developer may
 
 For more details about API versions, see :ref:`api-versions`.
 
-.. _api-version-upgrade-guide-630:
+.. _api-version-upgrade-guide-700:
 
-API version 630
+API version 700
```

Comment:
This one shouldn't change, as it's documenting the behavior changes of API version 630. We could go ahead and add a section now for API version 700 in addition to API version 630 if you wanted.